packages/core/src/utils/__tests__/did.test.ts

```diff
@@ -149,4 +181,16 @@ describe('Utils | Did', () => {
       expect(isDidIdentifier(didIdentifier)).toBe(false)
     })
   })
+
+  describe('getIndyDidFromVerficationMethod()', () => {
+    test.each(verificationMethod)('returns indy did from verification method', (method) => {
+      expect(typeof getIndyDidFromVerficationMethod(method)).toBe('string')
+    })
```

Comment:
```suggestion
    test.each(verificationMethod)('returns indy did from verification method', (method) => {
      expect(getIndyDidFromVerficationMethod(method)).toBe(method.publicKeyBase58)
    })
```

Reply:
@TimoGlastra After doing above change Test is failed because `getIndyDidFromVerficationMethod` returns the indy did and we try to compare with the public key

Reply:
Ah in that case we should test that it returns the indy did (so transform it manually once and then expect that as the return value. Otherwise we're not really testing that this method is extracting the indy did, it just tests whether it returns a string (which could be anything)